flake.nix

```diff
@@ -290,6 +293,17 @@
         nixpkgs.overlays = [ self.overlay nix.overlay ];
       };
 
+      runHydra = pkgs.callPackage ./run-hydra.nix {};
+      devShell = pkgs.hydra.overrideAttrs (old: {
```

Comment:
I think this is not a correct `devShell` and `nix flake check` will probably complain, since `devShell` needs to be a set of dev shells per platform (i.e. it should be `devShell.x86_64-linux`).

Reply:
And `devShell` shouldn't be necessary since `nix dev-shell` uses `defaultPackage` by default.

Reply:
As noted above i'll go ahead and drop the `devShell` and adjust the default shell expression accordingly.